scripts/devtron-reference-helm-charts/deployment-chart_4-19-0/values.yaml

```diff
@@ -702,3 +708,4 @@ tolerations: []
 imagePullSecrets: []
   # - test1
   # - test2
+persistentVolumeClaim: {}
```

Comment:
add the all the required key: values user need to pass in persistentVolumeClaim and make change in app-values.yaml also

Reply:
As suggested we will hide the values from the user and adding the values in readme